src/data/set/pairwise.lean

```diff
@@ -385,6 +385,34 @@ noncomputable def bUnion_eq_sigma_of_disjoint {s : set ι} {f : ι → set α}
 (equiv.set_congr (bUnion_eq_Union _ _)).trans $ Union_eq_sigma_of_disjoint $
   λ ⟨i, hi⟩ ⟨j, hj⟩ ne, h hi hj $ λ eq, ne $ subtype.eq eq
 
+lemma pairwise_disjoint_image_right_iff {f : α → β → γ} {s : set α} {t : set β}
```

Comment:
can you add a docstring to this lemma? The statement is not trivial to decipher.

Reply:
Do you like this one? If you draw a picture, it's obvious, but I feel like I'm missing some terminology here.